packages/server/src/fhir/operations/dbstats.ts

```diff
@@ -15,27 +15,57 @@ const operation: OperationDefinition = {
   system: true,
   type: false,
   instance: false,
-  parameter: [{ use: 'out', name: 'tableString', type: 'string', min: 1, max: '1' }],
+  parameter: [
+    {
+      use: 'in',
+      name: 'tableNames',
+      type: 'string',
+      min: 0,
+      max: '1',
+    },
+    {
+      use: 'out',
+      name: 'tableString',
+      type: 'string',
+      min: 1,
+      max: '1',
+    },
+  ],
 };
 
-export async function dbStatsHandler(_req: FhirRequest): Promise<FhirResponse> {
+export async function dbStatsHandler(req: FhirRequest): Promise<FhirResponse> {
   requireSuperAdmin();
 
+  const params = parseInputParameters<{ tableNames?: string }>(operation, req);
+
   const client = getDatabasePool(DatabaseMode.WRITER);
   const sql = `SELECT * FROM (
     SELECT
       i.relname AS table_name,
-      pg_total_relation_size(relid) AS raw_size,
-      pg_size_pretty(pg_total_relation_size(relid)) AS total_size,
-      pg_size_pretty(pg_relation_size(relid)) AS table_size,
-      pg_size_pretty(pg_indexes_size(relid)) AS all_indexes_size,
+      pg_total_relation_size(i.relid) AS raw_size,
+      pg_size_pretty(pg_total_relation_size(i.relid)) AS total_size,
+      pg_size_pretty(pg_relation_size(i.relid)) AS table_size,
+      pg_size_pretty(pg_indexes_size(i.relid)) AS all_indexes_size,
       reltuples::bigint AS estimated_row_count,
       indexrelname AS index_name,
       pg_size_pretty(pg_relation_size(indexrelid)) AS index_size,
-      idx_scan AS index_scans,
-      idx_tup_read AS index_entries_read,
-      idx_tup_fetch AS index_rows_fetched
-    FROM pg_stat_user_indexes i JOIN pg_class c ON i.relid = c.oid
+      i.idx_scan AS index_scans,
+      i.idx_tup_read AS index_entries_read,
+      i.idx_tup_fetch AS index_rows_fetched,
+      c.reloptions AS table_level_overrides,
+      n_live_tup AS live_tuples,
+      n_dead_tup AS dead_tuples,
+      last_autovacuum,
+      last_autoanalyze
+    FROM pg_stat_user_indexes i JOIN pg_class c ON i.relid = c.oid JOIN pg_stat_user_tables u ON i.relname = u.relname
+    ${
+      params.tableNames
```

Comment:
Can we move this to SQL params rather than string concatenation?  I realize this is a super admin endpoint, so the risk is relatively low, but more of a good hygiene thing.

Or, at the very least, some basic string sanitization

Reply:
FYI you can use `WHERE <column> = ANY($1::text[])` to compare against parameterized arrays